packages/cli/src/lib/index.js

```diff
@@ -96,11 +124,15 @@ export const generateTemplate = (templateFilename, { name, root, ...rest }) => {
   // We format .js and .css templates, we need to tell prettier which parser
   // we're using.
   // https://prettier.io/docs/en/options.html#parser
-  const parser = {
+  let parser = {
     '.css': 'css',
     '.js': 'babel',
   }[path.extname(templateFilename)]
 
+  if (path.basename(templateFilename).endsWith('sdl.js.template')) {
+    parser = 'babel'
+  }
+
```

Comment:
This hurt to write. Perhaps some one can advise on a better approach here? I am uncertain as to what other extends this is called, but from the `path.extname(templateFileName)` always returns `.template` when `scaffolding`